activerecord/lib/active_record/persistence.rb

```diff
@@ -357,10 +357,11 @@ def increment(attribute, by = 1)
     # +update_counters+, see that for more.
     # Returns +self+.
     def increment!(attribute, by = 1, touch: nil)
-      increment(attribute, by)
-      change = public_send(attribute) - (attribute_in_database(attribute.to_s) || 0)
-      self.class.update_counters(id, attribute => change, touch: touch)
-      clear_attribute_change(attribute) # eww
+      attribute_name = attribute_aliases.fetch(attribute.to_s) { attribute }.to_s
```

Comment:
@sgrif is there a nicer way to spell this? There's this: https://github.com/rails/rails/blob/5949cc05d395f5e1435d04d4adee7b26f13a517d/activerecord/lib/active_record/attribute_methods/read.rb#L55-L61

.. but the whole alias-expansion seems like something that could/should be wrapped into a single `String -> String` call.